src/binaryen-c.cpp

```diff
@@ -70,6 +71,11 @@ Literal fromBinaryenLiteral(BinaryenLiteral x) {
 static std::mutex BinaryenFunctionMutex;
 static std::mutex BinaryenFunctionTypeMutex;
 
+// Optimization options
+static int optimizeLevel = BINARYEN_DEFAULT_OPTIMIZE_LEVEL;
```

Comment:
this will have a single set of options for all Modules, globally. I guess that's ok, but we should document it.